compiler/passes/convert-typed-uast.cpp

```diff
@@ -6101,6 +6104,24 @@ bool TConverter::enter(const Conditional* node, RV& rv) {
 
   // Not param-known condition; visit both branches as normal.
 
+  auto insertCondTest = [&](types::QualifiedType condType,
```

Comment:
We should probably make a little widget / helper for this instead rather than hide it in a closure (later).